vmdb/app/views/layouts/quadicon/_host.html.haml

```diff
@@ -0,0 +1,58 @@
+- width = size == 150 ? 54 : 35
+- height = width
+- if @settings[:quadicons][:host]
+  .flobj
+    %img{:src => "/images/icons/#{size}/base.png", :width => size, :height => size}
+  - # Get the VM count %>
+  - vcount = item.vms.size
+  .flobj{:class => "a#{size}"}
+    %p
+      = vcount
+
+  - unless item.state.blank?
+    .flobj{:class => "b#{size}"}
+      %img{:src => "/images/icons/72/currentstate-#{h(item.state.downcase)}.png"}
+
+  .flobj{:class => "c#{size}"}
+    %img{:src => "/images/icons/new/vendor-#{h(item.vmm_vendor.downcase)}.png"}
+  .flobj{:class => "d#{size}"}
+    - case item.authentication_status
```

Comment:
@martinpovolny  this doesn't really works well with HAML and also breaks readability